Sources/PayPalDataCollector/DeviceInspectorProtocol.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+protocol DeviceInspectorProtocol {
+    func paypalDeviceIdentifier() -> String
```

Comment:
```suggestion
    func payPalDeviceIdentifier() -> String
```

Reply:
I can roll with that. I've seen a few variations when the first p is lowercase. The camel casing makes it more predictable though I can change this.